### ISSUE TITLE
Contact Journal: Suggest Entries from Calendar and Contacts

### ISSUE THREAD

**Issue by sedrubal** [corona-warn-app/cwa-wishlist#712]
## Feature description

To make it easier to create entries in the contact journal the CWA could suggest peoples names and phone numbers from the address book app and events/locations from the calendar app.

## Problem and motivation

For people who use the contact app of the system creating people entries in CWA is quite redundant. Also transferring locations / events from the calendar app to the CWA is too much manual work right now.

## Is this something you're interested in working on

No

---
Internal Tracking ID: [EXPOSUREAPP-11436](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11436)

**Comment by Ein-Tim** [comment]
Hi @sedrubal 👋

Thanks for this proposal! We have a very similar issue already; https://github.com/corona-warn-app/cwa-wishlist/issues/306, maybe you could take a look st it and decide whether this new issue is necessary or #306 covers your suggestions? 

Thanks and have a nice Sunday!

**Comment by sedrubal** [comment]
Oh I'm sorry, I didn't find that one. Yes it is basically the same with the difference, that #306 is limited to iOS and only to link contacts, not events.
I'm ok with closing this issue in favor of the other one but I'd suggest having the discussion for both iOS and Android.

**Comment by Ein-Tim** [comment]
@sedrubal 

Yes, you're right, maybe it makes sense to also leave this issue open. However, as you probably saw in #309, Apple & Google don't permit apps using their Exposure Notification Framework to ask for access to the users contacts.

**Comment by larswmh** [comment]
Thanks for your suggestion @sedrubal. We have created an internal ticket for it and will raise this topic internally.
Internal Tracking ID: [EXPOSUREAPP-11436](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11436)

---
Corona-Warn-App Open Source Team

**Comment by kroerig** [comment]
> @sedrubal
> 
> Yes, you're right, maybe it makes sense to also leave this issue open. However, as you probably saw in #309, Apple & Google don't permit apps using their Exposure Notification Framework to ask for access to the users contacts.

And what about the calendar. Are these apps allowed to import events?
